pkg/metrics/metrics.go

```diff
@@ -253,8 +259,49 @@ var (
 		Subsystem: Datapath,
 		Name:      "errors_total",
 		Help:      "Number of errors that occurred in the datapath or datapath management",
-	},
-		[]string{LabelDatapathArea, LabelDatapathName, LabelDatapathFamily})
+	}, []string{LabelDatapathArea, LabelDatapathName, LabelDatapathFamily})
+
+	// ConntrackGCDeleted is the number of deleted entries in the
+	// conntrack table that has expired.
+	ConntrackGCDeleted = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Subsystem: Datapath,
+		Name:      "conntrack_gc_deleted",
+		Help:      "Number of deleted entries in the conntrack table by the garbage collector process",
+	}, []string{LabelDatapathFamily, LabelScope})
+
+	// ConntrackGCRuns is the number of times that the conntrack GC
+	// process was run.
+	ConntrackGCRuns = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Subsystem: Datapath,
+		Name:      "conntrack_gc_runs_total",
+		Help:      "Number of times that the conntract garbage collector process was run",
+	}, []string{LabelDatapathFamily, LabelCompleted})
+
+	// ConntrackGCKeyFallbacks number of times that the conntrack key fallback was invalid.
+	ConntrackGCKeyFallbacks = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Subsystem: Datapath,
+		Name:      "conntrack_gc_key_fallbacks",
+		Help:      "Number of times that the Key fallback was invalid.",
```

Comment:
This was still unclear to me, in light of what tgraf said. Could we change it to "Number of times a key fallback was needed when iterating over the BPF map"